### PR TITLE
Icons and layouts.py changes for hive artist ui

### PR DESCRIPTION
**Pull request by keenfoong**
Wip Icons and layouts to get ready for the hive artist ui

### CODE REVIEW

File: zoo/libs/naming/naming.py [hunk 1]
```diff
@@ -104,6 +104,11 @@ def tokenValue(self, name):
             return self.config["tokens"][name]["value"]
         return self.config["tokens"][name]["default"]
 
+    def tokenValues(self, token):
+        ret = self.config["tokens"][token].copy()
+        del ret['default']
```
Comment (dsparrow27) [inline zoo/libs/naming/naming.py]: Probably want to remove the "value" key as well

File: zoo/libs/pyqt/widgets/layouts.py [hunk 2]
```diff
@@ -59,7 +59,7 @@ def text(self):
 class ComboBox(QtWidgets.QWidget):
     itemChanged = QtCore.Signal(int, str)
 
-    def __init__(self, label, items, parent=None):
+    def __init__(self, label="", items=[], parent=None):
```
Comment (dsparrow27) [inline zoo/libs/pyqt/widgets/layouts.py]: Shouldn't use mutable default arguments. 
http://docs.python-guide.org/en/latest/writing/gotchas/

File: zoo/libs/pyqt/widgets/layouts.py [hunk 3]
```diff
@@ -291,6 +303,35 @@ def connections(self):
         self.cancelBtn.clicked.connect(self.CancelBtnPressed.emit)
 
 
+class HRadioButtonGroup(QtWidgets.QWidget):
+    def __init__(self, radioList=[], default=0, parent=None):
```
Comment (dsparrow27) [inline zoo/libs/pyqt/widgets/layouts.py]: Same here for mutable args

File: zoo/libs/pyqt/widgets/layouts.py (hunk 2 above)
Comment (keenfoong) [reply zoo/libs/pyqt/widgets/layouts.py]: Its interesting behaviour that the same list is used from when the function is first created... I'll make these changes.

File: zoo/libs/naming/naming.py (hunk 1 above)
Comment (keenfoong) [reply zoo/libs/naming/naming.py]: Not sure... but I dont see a 'value' key in config.json (zoo/libs/naming/config.json), is it for if name is counter? (like in tokenValue)

Comment (dsparrow27) [reply zoo/libs/naming/naming.py]: Yeah that's right, i need to get around to updating the design of the config to better handle special types

**Pull request closed without merging**